server.py

```diff
@@ -1,5 +1,6 @@
 import os
 from holmes.utils.cert_utils import add_custom_certificate
+from holmes.utils.robusta import load_robusta_api_key
```

Comment:
same here, I would move the import after loading the custom cert

Reply:
done